File: app/src/main/java/com/originb/inkwisenote/commonutils/DateTimeUtils.java
package com.originb.inkwisenote.commonutils;

import java.time.Instant;
import java.time.ZoneId;
import java.time.format.DateTimeFormatter;

public class DateTimeUtils {
    public static String msToDateTime(long createdTimeMillis) {
        Instant instant = Instant.ofEpochMilli(createdTimeMillis);
        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")
                .withZone(ZoneId.systemDefault());

        return formatter.format(instant);
    }
}

File: app/src/main/java/com/originb/inkwisenote2/common/DateTimeUtils.kt
package com.originb.inkwisenote2.common

import java.time.Instant
import java.time.ZoneId
import java.time.format.DateTimeFormatter

object DateTimeUtils {
    fun msToDateTime(createdTimeMillis: Long): String {
        val instant = Instant.ofEpochMilli(createdTimeMillis)
        val formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")
            .withZone(ZoneId.systemDefault())

        return formatter.format(instant)
    }
}
